OJS.Workers.ExecutionStrategies/CSharp/CSharpProjectTestsExecutionStrategy.cs

```diff
@@ -155,7 +157,7 @@ public void RedirectConsoleOutputBeforeEveryTest()
 
         protected void SaveSetupFixture(string directory)
         {
-            this.SetupFixturePath = $"{directory}\\{SetupFixtureFileName}{Constants.CSharpFileExtension}";
+            this.SetupFixturePath = $"{directory}\\{SetupFixtureFileName}{CSharpFileExtension}";
```

Comment:
`Path.DirectorySeparatorChar`?

Reply:
This is change that is not part of this issue. Will be "lazy load refactored" when this strategy is modified or needed in Linux environment.